cardano-smash-server/src/Cardano/SMASH/Server/Config.hs

```diff
@@ -22,25 +31,32 @@ data SmashServerParams = SmashServerParams
   { sspSmashPort :: !Int
   , sspConfigFile :: !FilePath -- config is only used for the logging parameters.
   , sspAdminUsers :: !(Maybe FilePath)
+  , sspSmashPool :: !Int
   }
 
 defaultSmashPort :: Int
 defaultSmashPort = 3100
 
+defaultSmashPool :: Int
+defaultSmashPool = 5
```

Comment:
Arbitrarily chosen value here @kderme it really depends on how the postgres instance has been setup and how many connections it can handle. I would expect admins to override this value when setting it up.

Reply:
:+1: